crates/subspace-runtime/tests/integration/object_mapping.rs

```diff
@@ -1,16 +1,20 @@
 use codec::Encode;
 use frame_support::sp_io;
+use hex_literal::hex;
+use pallet_feeds::feed_processor::FeedProcessor;
+use pallet_feeds::{FeedConfig, FeedConfigs};
 use sp_consensus_subspace::runtime_decl_for_SubspaceApi::SubspaceApi;
-use subspace_core_primitives::crypto;
+use std::marker::PhantomData;
 use subspace_core_primitives::objects::BlockObjectMapping;
-use subspace_runtime::{Block, Call, Feeds, Header, Origin, Runtime, System, UncheckedExtrinsic};
+use subspace_core_primitives::Sha256Hash;
+use subspace_runtime::{
+    Block, Call, FeedProcessorId, GrandpaValidator, Header, PolkadotLike, Runtime, System,
+    UncheckedExtrinsic,
+};
 
 #[test]
 fn object_mapping() {
-    let data0: Vec<u8> = (0..=99).collect();
-    let data1: Vec<u8> = (123..=255).collect();
-    let data2: Vec<u8> = (0..=99).rev().collect();
-    let data3: Vec<u8> = (123..=255).rev().collect();
+    let (init_data, key, object) = get_encoded_blocks();
```

Comment:
This is an incorrect change. It uses the same object for all extrinsics that doesn't handle potential edge cases in case implementation does mapping incorrectly. Please revert it back (it should use default non-Polkadot feed processor) and add another case in case you need it.